app/controllers/attendees_controller.rb

```diff
@@ -0,0 +1,18 @@
+class AttendeesController < ApplicationController
+  before_action :logged_in_user
+
+  def attend
```

Comment:
It is better to use conventions to name the controller's actions, for example, if you are going to attend an event, it sounds like you want to `create` something.